app/src/utils/FakeMetadataMap.ts

```diff
@@ -64,26 +82,42 @@ const FakeMetadataMap: MetadataMap = {
       },
     ],
   },
-  diabetes: {
-    id: "diabetes",
-    name: "Diabetes Prevalence",
+  {
+    id: "brfss_diabetes",
+    name: "Diabetes Prevalence by state and race",
     data_source_name: "CDC's BrFSS",
     data_source_link: "https://gis.cdc.gov/grasp/diabetes/DiabetesAtlas.html#",
-    geographic_level: "State, County",
-    demographic_granularity: "Race/ethnicity, age, gender at county",
+    geographic_level: "State",
+    demographic_granularity: "Race/ethnicity",
     update_frequency: "?",
     update_time: "?",
     description:
       "Description placeholder for Diabetes Prevalence. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed id lectus urna. Vestibulum lobortis ac quam vel tincidunt. Quisque ex erat, efficitur nec sagittis vitae, dictum semper arcu. Vivamus metus felis, fringilla sit amet metus nec, feugiat suscipit nisl. Morbi posuere mi sit amet elit posuere, id gravida ligula facilisis. Aenean gravida a eros fringilla venenatis.",
     fields: [
```

Comment:
I also want FIPS here but we can address later. I actually manually added it myself in my PR so I could add it

Reply:
Yeah we definitely want FIPS codes in the datasets. That's something I'll mention to Maya and Jennie when we meet to discuss the data server

Reply:
Feel free to add it yourself though if you need it